src/OfficeConverter/OfficeConverter.php

```diff
@@ -140,7 +140,7 @@ protected function makeCommand($outputDirectory, $outputExtension)
     protected function prepOutput($outdir, $filename, $outputExtension)
     {
         $DS = DIRECTORY_SEPARATOR;
-        $tmpName = str_replace($this->extension, '', $this->basename).$outputExtension;
+        $tmpName = ($this->extension ? str_replace($this->extension, '', $this->basename) : $this->basename . '.').$outputExtension;
```

Comment:
PHP-CS-Fixer disallow space between concatenation.
So, please run `./vendor/bin/php-cs-fixer fix --diff --verbose src`
or replace your line by
```
$tmpName = ($this->extension ? str_replace($this->extension, '', $this->basename) : $this->basename.'.').$outputExtension;
```

Reply:
Done for this.